_posts/2023-11-30-indico-3-3-news.md

```diff
@@ -0,0 +1,88 @@
+  <img src="/assets/2023-11-30-indico-3-3-news/app2.png" />
+  <img src="/assets/2023-11-30-indico-3-3-news/app3.png" />
+</div>
+
+## 🧾 Receipts
+
+A long awaited feature: Indico now supports the generation of payment **receipts**. You can generate a fully customizable receipt for any payment made through Indico. It will be generated as a PDF and can be downloaded from the "payment details" page. <!-- add an extra use-case for this? (i.e. using this feature to create attendance certificates?) -->
+
+![](/assets/2023-11-30-indico-3-3-news/receipts1.png)
+![](/assets/2023-11-30-indico-3-3-news/receipts2.png)
+
+## 📅 "By Weekday" Recurring Bookings
+
+The Room Booking module now supports **recurring bookings that repeat on specific weekdays**. For example, you can now create a recurring booking that repeats every Monday, Wednesday, and Friday over a set period of time.
+
+![](/assets/2023-11-30-indico-3-3-news/recurring-booking.png)
+
+## ⚖️ Privacy improvements (GDPR compliance)
+
+Users can now be anonymized in Indico; this means that all personal identifiers associated with a user will be removed from Indico, whilst only keeping the data that is required for Indico to function properly, in an anonymized manner. In addition to this, users can now export a copy of all their personal data stored in Indico (data portability).
+
+![](/assets/2023-11-30-indico-3-3-news/export.png)
+
+<!-- Holding this back for now, since this feature isn't merged yet...
+## 🎟️ Google Wallet Integration
+
+Registration tickets can now be saved directly to Google Wallet. This allows users to easily access their tickets on their mobile device, without having to download and print the ticket (where applicable).
+
+> contribution "Community Contribution"
+> Feature kindly contributed by the **United Nations Office at Geneva**
+-->
+
+## 🇮🇹 Ciao Indico!
+
+Italian has been added as a new language in Indico. This was done thanks to a team of volunteer translators who worked hard to make Indico even more familiar to the more than 65 million italian speakers around the world. We promise the translation is pineapple-free.
+
+Want to help us translate Indico into your language? [Learn more here](https://docs.getindico.io/en/stable/installation/translations/#contributingo).
+
+## 📈 Other improvements and small features
+- Paper Editors are now allowed to edit their review comments
+- Links can now be added automatically, based on patterns in notes/minutes (e.g. issue trackers, GitHub repos etc.) <!-- wonder if this one isn't big enough to deserve its own section/screenshots? --> <!-- I agree -->
+- Improve the login page UI, allow overriding the logo URL config option and using custom logos for authentication providers
+
+> contribution "Community Contribution"
+> Feature kindly contributed by the **United Nations Office at Geneva**
+
+- Support for generating tickets and badges for each of the registrant's accompanying persons
+<!-- add more crap here --> 
```

Comment:
hey, our features aren't crap! 💩 😂

Reply:
Yeah sorry, it was me when I was writing the draft :trollface: